### PR TITLE
Added support for SWIFT associations

### PR DESCRIPTION
**Pull request by andykuszyk**
Relates to https://github.com/form3tech/swift-gateway/issues/25

### CODE REVIEW

File: client/associations/associations_client.go [hunk 1]
```diff
@@ -678,6 +688,40 @@ func (a *Client) DeleteStarlingID(params *DeleteStarlingIDParams) (*DeleteStarli
 	panic(msg)
 }
 
+/*
+  DeleteSwiftID deletes organisation s w i f t association
+*/
+func (a *Client) DeleteSwiftID(params *DeleteSwiftIDParams) (*DeleteSwiftIDNoContent, error) {
+	// TODO: Validate the params before sending
```
Comment (hallabro) [inline client/associations/associations_client.go:695]: Is this `TODO` still valid?

Comment (hallabro) [reply client/associations/associations_client.go:695]: I just noticed most similar functions has got the same comment.

File: swagger.yaml [hunk 2]
```diff
@@ -6809,6 +6809,177 @@ paths:
...
+          required: false
+          type: array
+          items:
+            type: string
+            format: uuid
+        - name: page[number]
+          in: query
+          description: Which page to select
+          required: false
+          type: string
```
Comment (hallabro) [inline swagger.yaml:6860]: `swagger validate` gave this warning message:

`WARNING: validation keywords of parameter "page[number]" in path "/swift" don't match its type string`

is this something we could fix?

**Review by hallabro — APPROVED** [review]

File: swagger.yaml (hunk 2 above)
Comment (andykuszyk) [reply swagger.yaml:6860]: I assume this is a result of `page[number]` being used for strings like `first` and numbers like `1`.

**Pull request merged**